src/main/java/core/basesyntax/controller/AddCarController.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax.controller;
+
+import core.basesyntax.lib.Injector;
+import core.basesyntax.model.Car;
+import core.basesyntax.model.Manufacturer;
+import core.basesyntax.service.CarService;
+import core.basesyntax.service.ManufacturerService;
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class AddCarController extends HttpServlet {
+    private static final String PACKAGE_NAME = "core.basesyntax";
```

Comment:
i don't think we need this constant. Also, you are creating the same constant in each controller.

Reply:
done